bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/ArrayValue.java

```diff
@@ -73,7 +74,7 @@
  * 
  * @since 0.995.0
  */
-public class ArrayValue implements RefValue, CollectionValue {
+public class ArrayValue implements RefValue, CollectionValue, BArray {
```

Comment:
Do we need to implement `BRefValue` and `CollectionValue`, since `BArray` interface already extending from them?

Reply:
We need to extend from RefValue since BRefValue has subset of methods from RefValue.